### PR TITLE
Demo run image extension

### PR DESCRIPTION
**Pull request by natalieparellano**

### CODE REVIEW

File: content/docs/extension-author-guide/create-extension/setup-local-environment.md [hunk 1]
```diff
@@ -69,7 +72,11 @@ We unset any custom lifecycle image that may have been provided to ensure that t
 
 <!-- test:exec -->
 ```bash
-git clone https://github.com/buildpacks/samples.git
+# FIXME: remove branch when https://github.com/buildpacks/samples/pull/149 is merged
+git clone https://github.com/buildpacks/samples.git && \
+  cd samples && \
```
Comment (itsdarshankumar) [inline content/docs/extension-author-guide/create-extension/setup-local-environment.md:77]: shouldn't we also add git fetch origin runext/update-samples command

Comment (natalieparellano) [reply content/docs/extension-author-guide/create-extension/setup-local-environment.md]: Added in 79ab9738

**Comment by natalieparellano** [comment]
Note: the tests for this PR are going to continue to fail until pack 0.30.0 exists 😿

**Comment by natalieparellano** [comment]
@AidanDelaney do you approve this one?

**Review by AidanDelaney — APPROVED** [review]
LGTM

**Pull request merged**